certora/helpers/Utils.sol

```diff
@@ -3,7 +3,7 @@
 pragma solidity >=0.8.0;
 
 contract Utils {
-    function to_bytes4(uint32 x) public pure returns (bytes4) {
+    function toBytes4(uint32 x) public pure returns (bytes4) {
         return bytes4(x);
     }
```

Comment:
CVL2 should be able to handle this no ?

Reply:
Unfortunately no, I tried it and it failed with:
`Only number literals may be used in bytes conversion`

Also the names that are prefixed by `to_` are reserved now, which is why I needed to rename this function

Reply:
ok ok